storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -164,13 +164,13 @@
 (def COMMON-FIELDS [:emitted :transferred])
 (defrecord CommonStats [emitted transferred rate])
 
-(def BOLT-FIELDS [:acked :failed :process-latencies :executed :execute-latencies])
+(def BOLT-FIELDS [:acked :failed :process-latencies :executed :execute-latencies :deserialize-time])
 ;;acked and failed count individual tuples
-(defrecord BoltExecutorStats [common acked failed process-latencies executed execute-latencies])
+(defrecord BoltExecutorStats [common acked failed process-latencies executed execute-latencies deserialize-time])
 
-(def SPOUT-FIELDS [:acked :failed :complete-latencies])
+(def SPOUT-FIELDS [:acked :failed :complete-latencies :deserialize-time])
```

Comment:
Are we even collecting de-serialize time for the spout? I didn't see it anywhere.